api/api_ethereum.go

```diff
@@ -793,8 +793,8 @@ func newEthRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNum
 		result.GasFeeCap = (*hexutil.Big)(tx.GasFeeCap())
 		result.GasTipCap = (*hexutil.Big)(tx.GasTipCap())
 		// TODO-Klaytn: If we change the gas price policy from fixed to dynamic,
-		// we should change the params.BaseFee(fixed value) to dynamic value.
-		price := math.BigMin(new(big.Int).Add(tx.GasTipCap(), new(big.Int).SetUint64(params.BaseFee)), tx.GasFeeCap())
+		// we should change the params.ZeroBaseFee(fixed value) to dynamic value.
+		price := math.BigMin(new(big.Int).Add(tx.GasTipCap(), new(big.Int).SetUint64(params.ZeroBaseFee)), tx.GasFeeCap())
```

Comment:
In this case, it seems to be necessary to return the actual baseFee used when this transaction is actually processed.